customTests/azure_ib_write_bw.nhc

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+IB_WRITE_BW_EXE_PATH=/opt/azurehpc/test/nhc/perftest-4.5/ib_write_bw
+IB_WRITE_BW=`basename $IB_WRITE_BW_EXE_PATH`
+IB_WRITE_BW_DURATION=10
+IB_WRITE_BW_ARGS="-s $(( 1 * 1024 * 1024 )) -D ${IB_WRITE_BW_DURATION} -x 0 -F --report_gbits -d mlx5_ib0"
+SLEEP_TIME=5
+
+HOSTNAME=`hostname`
+
+function check_ib_bw(){
+
+EXP_IB_BW=$1
+
+IB_WRITE_BW_OUT1=$( taskset -c 0 $IB_WRITE_BW_EXE_PATH $IB_WRITE_BW_ARGS  > /dev/null &)
```

Comment:
is taskset better than using numactl? With numactl you can add --membind, although I'm not sure how important that is for this test.

Reply:
let me test this out to see if I see a difference. If not I am inclined to leave it alone. ALso, I think Numa mapping becomes more important when there is more than one IB nic. These tests are for one HB series which only have one IB Nic.

Reply:
I changed it to numactl --membind=0. and its working as intended

Reply:
Fixed per off line discussion